tests/models/test_llamacpp.py

```diff
@@ -0,0 +1,7 @@
+from outlines.fsm.guide import create_states_mapping
+from outlines.models.llamacpp import LlamaCppTokenizer
+
+
+def test_create_states_mapping_llamacpp(model):
+    """Minimal reproducer for #922, error passing llamacpp tokenizer to create_states_mapping"""
+    create_states_mapping("a", LlamaCppTokenizer(model.model))
```

Comment:
We shouldn't need this anymore.  Plus, our tests need to at least `assert` something (excluding rare cases in which that's not possible/practical, of course).

Reply:
To clarify, this test fails in `main` because `create_states_mapping` cannot be passed a `LlamaCppTokenizer`. Will remove though, as the test you recommended is excellent for verifying the `LlamaCppTokenizer` changes.